drivers/wifi/nrf700x/zephyr/src/qspi/src/spi_if.c

```diff
@@ -222,6 +224,13 @@ int spim_init(struct qspi_config *config)
 
 	k_sem_init(&spim_config->lock, 1, 1);
 
+	if (freq_m > 8) {
```

Comment:
I'd suggest to use `(spi_spec.config.frequency > MHZ(8))` to avoid an unnecessary division here (if you use one only in the call to `LOG_INF()`, it will be cut out when logging is disabled).
And shouldn't it be consistent with the QSPI case, I mean `>= MHZ(16)`?

Reply:
Same as above, this was done only for checkpatch constant warning.